charts/portal/templates/portal-enterprise/portal-enterprise-deployment.yaml

```diff
@@ -137,11 +137,17 @@ spec:
           {{- if .Values.portalEnterprise.resources }}
           resources: {{- toYaml .Values.portalEnterprise.resources | nindent 12 }}
           {{- end }}
+          {{- if .Values.global.enableMetricEndpoints }}
```

Comment:
Can be change to 
ports:
   - containerPort: 8080
 {{- if .Values.global.enableMetricEndpoints }}
    - containerPort: 9999
  {{ end }}

Reply:
I just made a change.  Please validate @APIM-KiranVaddadi